operator/cmd/cilium_node.go

```diff
@@ -48,8 +48,11 @@ var (
 	ciliumNodeStore cache.Store
 
 	k8sCiliumNodesCacheSynced = make(chan struct{})
+	ciliumNodeManagerSynced   = make(chan struct{})
```

Comment:
I would suggest to either:
a) rename this to `nodeManagerQueueSynced`
b) rename `nodeManagerQueue` to `ciliumNodeManagerQueue` and `ciliumNodeManagerSynced` to `ciliumNodeManagerQueueSynced`

Reply:
@dylandreimerink I meant this one

Reply:
Went for option B, should be good now